mlflow/pipelines/steps/evaluate.py

```diff
@@ -124,7 +127,7 @@ def _run(self, output_directory):
         model_uri = get_step_output_path(
             pipeline_root_path=self.pipeline_root,
             step_name="train",
-            relative_path="model",
+            relative_path=TrainStep.MODEL_ARTIFACT_RELATIVE_PATH,
```

Comment:
Model artifact relative path needs to be referenced by other steps for model URL construction